docs/source/install/deb.rst

```diff
@@ -23,7 +23,7 @@ The easiest way to install these is to use the requirements.txt file from the |s
 
 ::
 
-    curl -q -k -O https://ops.stackstorm.net/releases/st2/0.9dev/requirements.txt
+    curl -q -k -O https://raw.githubusercontent.com/StackStorm/st2/master/requirements.txt
```

Comment:
This way we depend on the availability on Github, which I guess, is not that big of a deal since we already indirectly depend on that when installing Python dependencies (some if them are installed from a Github git repo).

Ideally, down the road we would move to self hosted wheels and in the short term at least our mirror of PyPi packages we use...